src/store/metadata/stub/content/editions/versions/pupils-at-expected-standards.json

```diff
@@ -0,0 +1,88 @@
+            ],
+            "@id": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions/1",
+            "identifier": "1",
+            "issued": "2023-03-31T12:15:00Z",
+            "title": "Pupils at the expected standards at the end of key stage 2",
+            "summary": "Percentage of pupils in state-funded schools meeting the expected standard in reading, writing and maths at the end of key stage 2, England (state-funded schools), academic years 2015/16 - 2021/22",
+            "description": "This dataset shows the percentage of pupils in state-funded schools meeting the expected standard in reading, writing and maths at the end of key stage 2, in England state-funded schools, for academic years 2015/16 - 2021/22.",
+            "download_url": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
+                    {
+                      "name": "period",
+                      "datatype": "string",
+                      "titles": "Period",
+                      "description": "The time period associated with the observation."
+                    },
+                    {
+                      "name": "areacd",
+                      "datatype": "string",
+                      "titles": "ONS Geography Code",
+                      "description": "The geography code associated with the observation."
+                    },
+                    {
+                      "name": "areanm",
+                      "datatype": "string",
+                      "titles": "Area Name",
+                      "description": "The name assiciated with The geography code "
+                    },
+                    {
+                      "name": "geography",
+                      "datatype": "string",
+                      "titles": "Geography Type",
+                      "description": "The categorisation of the geography code"
+                    },
+                    {
+                      "name": "value",
+                      "datatype": "decimal",
+                      "titles": "Geographic area with percentage of pulpils at expected standards by at least one provider",
+                      "description": "Observational Value"
+                    },
+                    {
+                      "name": "measure",
+                      "datatype": "string",
+                      "titles": "Measure Type",
+                      "description": "Measure Type of the Observational Value"
+                    },
+                    {
+                      "name": "unit",
+                      "datatype": "string",
+                      "titles": "Unit",
+                      "description": "Unit Type of the Observational Value"
+                    },
+                    {
+                      "name": "lower confidence interval (95%)",
```

Comment:
These names should be consistent with the other recently updated datasets: lower_confidence_interval_95 instead of lower confidence interval (95%)